pkg/api/v1/assignments_create.go

```diff
@@ -26,46 +28,63 @@ func (r *Router) assignmentsCreate(c echo.Context) error {
 		return err
 	}
 
-	assignments := models.AssignmentSlice{}
-
 	tx, err := r.db.Begin()
 	if err != nil {
 		r.logger.Errorw("error starting transaction", "error", err)
 		return v1InternalServerErrorResponse(c, err)
 	}
 
-	for _, p := range payload {
-		assignment := models.Assignment{
-			TenantID:       tenantID,
-			FrontendID:     p.FrontendID,
-			LoadBalancerID: p.LoadBalancerID,
-			PoolID:         p.PoolID,
+	assignment := models.Assignment{
+		TenantID:   tenantID,
+		FrontendID: payload.FrontendID,
+		PoolID:     payload.PoolID,
+	}
+
+	if err := assignment.Insert(ctx, tx, boil.Infer()); err != nil {
```

Comment:
I think you're only doing the one insert here?  is there an advantage to using a transaction?

Reply:
nope, 👍